helm/datalab/values.yaml

```diff
@@ -1,11 +1,13 @@
----
+# Default values for datalab.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
 namespace: test
 environment: test
 datalabName: testlab
 domain: test-datalabs.nerc.ac.uk
 kubernetesMasterHostName: datalabs-k8s-master
 
-datalabVersion: 0.31.1-119-g00739398
```

Comment:
LGTM, but just one thing. Wonder if it would be better to have "datalabVersion" in the "values.yaml" file? For example if I want to deploy to the prod, I would supply a custom "values.yaml" file with the values specific to the prod cluster and I can use a specific version of DataLabs I want in the prod cluster rather than changing "appVersion" in "Chart.yaml"..

Reply:
This is a good point. The reason I went the way I did is because I didn't want the app version being deployed to be different to the one specified in the Helm chart as I think that could cause some confusion. I agree that we are going to have to be able to deploy different versions to prod. 

We decided a couple of sprints ago that we were going to start tagging the main repo with release candidates for prod. Would it be good to also create a new chart at the point so it would be easy to deploy at a later date?

Reply:
Yes, that would be good